src/main/java/com/sngular/kloadgen/sampler/schemaregistry/adapter/impl/AvroSchemaAdapter.java

```diff
@@ -0,0 +1,42 @@
+package com.sngular.kloadgen.sampler.schemaregistry.adapter.impl;
+
+import java.util.List;
+import java.util.Map;
+
+import io.confluent.kafka.schemaregistry.avro.AvroSchema;
+import io.confluent.kafka.schemaregistry.client.rest.entities.SchemaReference;
+import org.apache.avro.Schema;
+
+public class AvroSchemaAdapter extends ConfluentParsedSchemaMetadata{
+
+  public static final String TYPE = "AVRO";
+
+  private Schema schemaObj;
+  private String canonicalString;
+  private Integer version;
+  private List<SchemaReference> references;
+  private Map<String, String> resolvedReferences;
+  private boolean isNew;
+
+  private transient int hashCode = NO_HASHCODE;
```

Comment:
Several of these are unused. You should also run this file through the formatter